src/opencmiss/utils/zinc/field.py

```diff
@@ -574,7 +575,7 @@ def find_or_create_field_stored_mesh_location(fieldmodule: Fieldmodule, mesh: Me
     if not name:
         name = "location_" + mesh.getName()
     field = fieldmodule.findFieldByName(name)
-    if field_exists(fieldmodule, name, 'StoredMeshLocation', mesh.getDimension()):
+    if field_exists(fieldmodule, name, 'StoredMeshLocation', 1):
```

Comment:
I think this deserves a comment as to why `1` is the proper value.

Reply:
Have added a comment to say 1 is correct.
FYI I removed the ability for non-numeric 'FE' fields to have multiple components about 10 years ago, as it was becoming too time consuming to maintain plus they don't really form vector quantities.